docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.9"
+services:
```

Comment:
you also need to add service to run your cart-api
And for cart api - you need to create docker file with multistage building